src/sentry/services/hybrid_cloud/notifications/impl.py

```diff
@@ -120,8 +121,23 @@ def get_subscriptions_for_projects(
         type: NotificationSettingEnum,
     ) -> Mapping[int, tuple[bool, bool, bool]]:
         """
-        Returns a mapping of project_id to a tuple of (is_disabled, is_active, has_only_inactive_subscriptions)
+        Deprecated: use subscriptions_for_projects instead.
         """
+        result = self.subscriptions_for_projects(
+            user_id=user_id, project_ids=project_ids, type=type
+        )
+        return {
+            project_id: (sub.is_disabled, sub.is_active, sub.has_only_inactive_subscriptions)
+            for project_id, sub in result.items()
+        }
+
+    def subscriptions_for_projects(
+        self,
+        *,
+        user_id: int,
+        project_ids: list[int],
+        type: NotificationSettingEnum,
+    ) -> Mapping[int, RpcSubscriptionStatus]:
```

Comment:
Something I've been wondering lately: would it be helpful to make these more complex return types into exportable types in the `service.py` file that other code can import just to cut down on the type repetition?

Reply:
> would it be helpful to make these more complex return types into exportable types in the service.py file that other code can import just to cut down on the type repetition?

I think it would be helpful to reduce noise and boilerplate types.